state_names.py

```diff
@@ -3,18 +3,23 @@
 State names in a dictionary
 File needs reformatting
 """
+
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
 CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
-for key in CODE_TO_NAME:
-    value = CODE_TO_NAME[key]
-    print(f"{key} is {value}")
 
+all_sentences = ""
 state_code = input("Enter short state: ").upper()
 for i in CODE_TO_NAME:
```

Comment:
this for loop require user enter at least 7 input.

Reply:
Yes these are committed and thus red blocks are not in the actual code now